xAPI.md

```diff
@@ -510,6 +513,69 @@ Clarifications and additional examples including:
 - Addiitional explanitory text and diagrams
 [1.0.2...1.0.3](https://github.com/adlnet/xAPI-Spec/compare/1.0.2...1.0.3)
 
+
+<a name="Appendix1B"/>
+### Appendix B: cmi5 Example 
+
+The following example illustrates a statement following the cmi5 Community of Practice. It demonstrates use of Extensions 
+and "category" Context Activities.
+
+```
+{
+  "id":"2a41c918-b88b-4220-20a5-a4c32391a240",
+  "actor": {
+    "objectType": "Agent",
+    "name": "Gert Frobe",
+    "account": {
+      "homePage": "http://example.adlnet.gov",
+      "name": "1625378"
+    }
+  },
+  "verb": {
+    "id": "http://w3id.org/xapi/verbs/failed",
```

Comment:
This verb isn't in cmi5? Would have to be `http://adlnet.gov/expapi/verbs/failed` at least until changes from https://github.com/AICC/CMI-5_Spec_Current/issues/353 land, and I'd say we are well away from that happening as it is a huge change.